src/Symfony/Component/OptionsResolver/OptionConfigurator.php

```diff
@@ -0,0 +1,130 @@
+<?php
+declare(strict_types=1);
```

Comment:
That's good, but we don't use this technique in Symfony repo, so this line should be removed.